### DISCUSSION TITLE
PaddleOCR/main/ppocr/blog/distributed_training

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#14322] [Q&A]
# PaddleOCR/main/ppocr/blog/distributed_training

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/main/ppocr/blog/distributed_training.html

**Comment by SidneyRey** [comment]
单机多卡训练时，我指定ips为本地：127.0.0.1，但后面仍然是连接自己检索的ip，这个ip被安全封掉连不上，怎么办？
启动命令：
python -m paddle.distributed.launch --devices 0,1 --log_dir /data/source/ocr/PaddleX-release-3.0-beta2/output/pubtab_t1/distributed_train_logs --ips 127.0.0.1 tools/train.py -c /home/source/ocr/PaddleX-release-3.0-beta2/tablerecmodel_SLANet.yml
相关日志：
I1204 18:02:08.401266 1124716 tcp_utils.cc:181] The server starts to listen on IP_ANY:61077
I1204 18:04:18.641605 1124716 tcp_utils.cc:107] Retry to connect to 10.218.108.187:61077 while the server is not yet listening.
I1204 18:06:31.761586 1124716 tcp_utils.cc:107] Retry to connect to 10.218.108.187:61077 while the server is not yet listening.

**Reply by SWHL** [reply]
看着是PaddleX使用问题，请移步：https://github.com/PaddlePaddle/PaddleX/discussions
